Wrappers/Python/test/testclass.py

```diff
@@ -60,3 +60,21 @@ def assertNumpyArrayEqual(self, first, second):
     def assertNumpyArrayAlmostEqual(self, first, second, decimal=6):
         np.testing.assert_array_almost_equal(first, second, decimal)
         
+    def assertNumpyArrayAllClose(self, first, second, decimal=6):
```

Comment:
I'm not sure when it's useful to use allclose versus almostequal... but should we specify the tolerance here rather than decimal?

Reply:
Good catch - this was incorrect and wasn't used so I have deleted it. 

The idea of using all close vs almost equal is that all close uses relative errors and almost equal uses a number of decimal place. For very large numbers or very small numbers relative errors is safer.